constraints/miden-vm/hash.air

```diff
@@ -202,4 +202,67 @@ ev hash_chiplet(main: [s[3], r, h[12], i]):
     enf hasher_state([s, h, i])
 
     # Multiset check constraints
-    # TODO: Apply multiset check constraints
\ No newline at end of file
+    # TODO: Apply multiset check constraints
+
```

Comment:
1. The new helper evaluators should go above in the helper evaluator section so that the `hash_chiplet` evaluator is still the last thing in the file as we've done in other places. 
2. We should add calls to the two new evaluators here and remove this TODO